ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/tasks/CompleteBlocksTask.java

```diff
@@ -79,7 +79,11 @@ private CompleteBlocksTask(
 
```

Comment:
The constructor needs to correct a block body with an empty list of withdrawals when there is an empty block Shanghai. This always creates a block body with an empty optional which is incorrect post-shanghai.

Reply:
Yes, we've changed the implementation to take protocolSchedule into account for the empty body creation Will push that change soon. Thanks!